ledger-core/v2/virtual/handlers/vdelegatedcallrequest_test.go

```diff
@@ -25,8 +24,11 @@ import (
 	"github.com/insolar/assured-ledger/ledger-core/v2/testutils/gen"
 	"github.com/insolar/assured-ledger/ledger-core/v2/testutils/slotdebugger"
 	"github.com/insolar/assured-ledger/ledger-core/v2/virtual/object"
+	"github.com/insolar/assured-ledger/ledger-core/v2/virtual/token"
 )
 
+var deadBeef = [...]byte{0xde, 0xed, 0xbe, 0xef}
```

Comment:
typo! :) should be {0xde, 0xad, 0xbe, 0xef}

Reply:
inDEED :D